.github/workflows/release.yml

```diff
@@ -429,6 +429,83 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  # Comment in a locked discussion that notifies about only `gitoxide` (e.g. not `gix-*`) releases.
+  announce-release:
+    runs-on: ubuntu-latest
+
+    needs: [ create-release, publish-release ]
+
+    permissions:
+      contents: write  # Needed to distinguish unpublished (still draft) from missing releases.
+      discussions: write
+
+    env:
+      REPOSITORY: ${{ github.repository }}
+      VERSION: ${{ needs.create-release.outputs.version }}
+      DISCUSSION_URL: ${{ vars.RELEASE_ANNOUNCEMENTS_URL }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      - name: Find the discussion ID
+        run: |
+          [[ "$DISCUSSION_URL" =~ ^https://github\.com/([^/:@]+)/([^/:@]+)/discussions/([0-9]+)$ ]]
+          owner="${BASH_REMATCH[1]}"
+          name="${BASH_REMATCH[2]}"
+          number="${BASH_REMATCH[3]}"
+
+          id="$(gh api graphql -f query='
+          query GetDiscussionId($owner: String!, $name: String!, $number: Int!) {
+            repository(owner: $owner, name: $name) {
+              discussion(number: $number) {
+                id
+              }
+            }
+          }' -F owner="$owner" -F name="$name" -F number="$number" --jq .data.repository.discussion.id)"
+
+          echo "DISCUSSION_ID=$id" >> "$GITHUB_ENV"
+
+      - name: Avoid announcing a test in a non-test thread
+        run: |
+          case "$VERSION" in
+          TEST-* | *-DO-NOT-USE )  # NOTE: Should be the same pattern as in `create-release` above.
+            echo "The release name indicates testing, so we'll only post if the thread is for that."
+            ;;
+          * )
+            is_draft="$(gh release --repo="$REPOSITORY" view "$VERSION" --json isDraft --jq .isDraft)"
+            if [ "$is_draft" = false ]; then
+              exit 0  # OK to post in a non-test announcement thread.
+            fi
+            echo "The release is not published, so we'll only post if the thread is for testing."
+            ;;
+          esac
+
+          title="$(gh api graphql -f query='
+          query($id: ID!) {
+            node(id: $id) {
+              ... on Discussion {
+                title
+              }
+            }
+          }' -F id="$DISCUSSION_ID" --jq .data.node.title)"
+
+          grep -Eiqz '^[[(]?test\b' <<<"$title"
+
+      - name: Post the comment
+        run: |
+          grep -Eqx '[[:alnum:]._+-]+' <<<"$VERSION"  # Ensure the version needs no sanitization.
+          release_url="https://github.com/$REPOSITORY/releases/tag/$VERSION"
+          comment_body="\`gitoxide\` [$VERSION]($release_url) has been released."
```

Comment:
As seen in https://github.com/EliahKagan/gitoxide/discussions/6, this creates comments like:

> `gitoxide` [v0.39.0-alpha.1-DO-NOT-USE](https://github.com/EliahKagan/gitoxide/releases/tag/v0.39.0-alpha.1-DO-NOT-USE) has been released.

But maybe this would look better:

> gitoxide [v0.39.0-alpha.1-DO-NOT-USE](https://github.com/EliahKagan/gitoxide/releases/tag/v0.39.0-alpha.1-DO-NOT-USE) has been released.

Or this:

> Gitoxide [v0.39.0-alpha.1-DO-NOT-USE](https://github.com/EliahKagan/gitoxide/releases/tag/v0.39.0-alpha.1-DO-NOT-USE) has been released.

Though I think that last way could be confusing because the `gitoxide` *crate* is never, in practice, written `Gitoxide`.

Reply:
I have been quite consistent with writing `gitoxide` but it's merely for it to stick out. Probably there is no need for that in all cases.

However, for now I think it's ok to stick the comment body as is and observe that it works when the release happens in a couple of days.